@here/harp-mapview/lib/MapView.ts

```diff
@@ -965,6 +998,17 @@ export class MapView extends THREE.EventDispatcher {
         this.m_forceCameraAspect = aspect;
     }
 
+    /**
+     * Maximum FPS. If defined (and > 0) it is the maximum FPS that is used.
+     */
+    set maxFps(fps: number) {
+        this.m_maxFps = Math.max(0, fps);
+    }
+
+    get maxFps(): number {
+        return this.m_maxFps > 0 ? this.m_maxFps : 0;
```

Comment:
Math.max(0, this.m_maxFps) would be equivalent?

Reply:
Yeah, I think so. As we already using max everywhere, we can also do it here.